run.sh

```diff
@@ -34,6 +34,9 @@ source ${scripts_absolute_dir}/deployment-actions-${DEPLOYMENT_MECHANISM}.sh
 deployment_action=${1:-"setup_everything"}
 
 case "$deployment_action" in
+    "deploy_network")
+        deploy_network
+        ;;
```

Comment:
This means it's not on the standard deploy path, which is fine for CI (which would reuse networks), and convenient for devs. :+1:

Reply:
I changed this. Why would CI reuse networks?

Reply:
Because there is no point of re-creating networks? I don't think we should have the overhead of creating networks, while it brings no positive value (it's not like we are making this multi tenant or anything). But I can be convinced otherwise.

Reply:
It's about creating multiple envs in parallel (as we discussed yesterday, we'll have a single variable (currently `PREFIX` but could/should be something like `ENV_NAME`). So deploying the network for the given `ENV_NAME` is needed during a standard deployment imo

Reply:
Well, we can create multiple environment in parallel while still keeping the efficiency of not creating networks. Although this feature is, for me, interesting. It's a convenience.

Reply:
The updated code now checks if there is already a network stack (it's not checking the network/subnet/router/gateway).
So if you have already a network stack (assuming that it is working correctly), then the creation will be skipped.
So it's possible to reuse the current stack now.

Reply:
Update: `os_stack` automatically creates (if stack does not exist) or update (if stack is already there) the stack.